tools/interop_matrix/testcases/python__master

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+echo "Testing ${docker_image:=grpc_interop_python:797ca293-94e8-48d4-92e9-a4d52fcfcca9}"
```

Comment:
It's surprising to me that fixing this problem involves adding this whole large file to the codebase - was this an inadvertent addition or does this need to be here?

Reply:
This is the test commands file that's needed for every language by the framework. Previously I was just testing locally.

Reply:
Well... okay.

Reply:
This is the current design. Here's the background: we debated on whether to create separate file for test cases for bake them into the images. Folks voted for the first, hence the need of these files checked in. But as I took over this project and started expansion, I felt the 2nd option probably works better. But it'll require a major architectural change.